### DISCUSSION TITLE
Storage Driver

### DISCUSSION THREAD

**Discussion by astolzen** [openshift/microshift#1005] [General]
Hi Team,

Why did you replace the simple "hostpath provisioner" from the initial project with "topo-lvm" ?
Hostpath runs 1 container, while topo-lvm ist much slower on simple edge hardware and much more complicated . It also runs a total of 8 containers. 
Edge-Devices typically have one sinlge SD-Card, so why bother with a LVM management for PVs instead of simple subdirectories ?

Andreas

**Comment by dhellmann** [comment]
You'll find some details about that decision in the enhancement document: https://github.com/openshift/enhancements/blob/master/enhancements/microshift/microshift-default-csi-plugin.md
